src/wizards.ts

```diff
@@ -0,0 +1,250 @@
+        if (name === 'value') {
+          newSink.textContent = diff.theirs;
+        } else if (diff.theirs === null) newSink.removeAttribute(name);
+        else newSink.setAttribute(name, diff.theirs);
+      actions.push({ old: { element: sink }, new: { element: newSink } });
+    }
+
+    let acted = false;
+
+    const selectedChildDiffs = (<ListItem[]>checkList.selected)
+      .filter(item => item.classList.contains('child'))
+      .map(item => childDiffs[(item.value as unknown) as number]);
+    if (selectedChildDiffs.length) {
+      for (const diff of selectedChildDiffs)
+        if (!diff.ours)
+          actions.push({
+            new: { parent: sink, element: diff.theirs, reference: null },
+          });
+        else if (!diff.theirs)
+          actions.push({
+            old: {
+              parent: sink,
+              element: diff.ours,
+              reference: diff.ours.nextElementSibling,
+            },
+          });
+        else {
+          acted = true;
+          wizard.dispatchEvent(
+            newWizardEvent(
+              mergeWizard(diff.ours, diff.theirs, {
+                ...options,
+                title: undefined,
+              })
+            )
+          );
+        }
+    }
+
+    if (actions.length === 0 && !acted) wizard.dispatchEvent(newWizardEvent());
+
+    return [
+      {
+        actions,
+        title: get('merge.log', {
+          sink: describe(sink),
+          source: describe(source),
+          tag: sink.tagName,
+        }),
+      },
+    ];
+  };
+}
+
+export function mergeWizard(
+  sink: Element,
+  source: Element,
+  options?: MergeOptions
+): Wizard {
+  const attrDiffs: [string, Diff<string>][] = [];
+  const ourText = sink.textContent ?? '';
+  const theirText = source.textContent ?? '';
+
+  if (
+    sink.childElementCount === 0 &&
+    source.childElementCount === 0 &&
+    theirText !== ourText
+  )
+    attrDiffs.push(['value', { ours: ourText, theirs: theirText }]);
+
+  const attributeNames = new Set(
+    source.getAttributeNames().concat(sink.getAttributeNames())
+  );
+
+  for (const name of attributeNames)
+    if (source.getAttribute(name) !== sink.getAttribute(name))
+      attrDiffs.push([
+        name,
+        <Diff<string>>{
+          theirs: source.getAttribute(name),
+          ours: sink.getAttribute(name),
+        },
+      ]);
+
+  const childDiffs: Diff<Element>[] = [];
+  const ourChildren = Array.from(sink.children);
+  const theirChildren = Array.from(source.children);
+
+  theirChildren.forEach(theirs => {
+    const twinIndex = ourChildren.findIndex(ourChild =>
+      isSame(theirs, ourChild)
+    );
+    const ours = twinIndex > -1 ? ourChildren[twinIndex] : null;
+
+    if (ours) ourChildren.splice(twinIndex, 1);
+    if (ours && isEqual(theirs, ours)) return;
+
+    if (!ours || !isEqual(theirs, ours)) childDiffs.push({ theirs, ours });
+  });
+
+  ourChildren.forEach(ours => childDiffs.push({ theirs: null, ours }));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195001)
```suggestion
  ourChildren.forEach((ours) => childDiffs.push({ theirs: null, ours }));
```